shared/charts/gpii-flushtokens/templates/cronjob.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  namespace: {{ .Release.Namespace | quote }}
+  name: {{ template "flushtokens.name" . }}
+spec:
+  schedule: "*/15 * * * *"
+  concurrencyPolicy: Allow
+  startingDeadlineSeconds: 200
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: {{ template "flushtokens.name" . }}
+        spec:
+          containers:
+          - name: gpii-flushtokens
+            image: "{{ .Values.image.repository }}@{{ .Values.image.checksum }}"
+            command: [ '/app/scripts/dockerDeleteExpiredAccessTokens.sh' ]
+            env:
+            - name: GPII_COUCHDB_URL
+              value: '{{ .Values.couchdbUrl }}'
+            - name: DELETE_ALL
+              value: ''
```

Comment:
@mrtyler, is there any restriction on how to specify the value of the DELETE_ALL flag in YAML?

- is the use of an empty string just a convention?
- or, is it a requirement?
 - or, could it be set to "false"?

Motivation: @cindyli suggested exploring this change in [PR 777](https://github.com/GPII/universal/pull/777#discussion_r290925828)

Reply:
The value here is consumed only by your script, so you may use whatever your script will understand. `''` and `false` are the same in `bash`'s eyes, but I agree that `false` is probably clearer than `''`.